sale_order_import_edifact/__manifest__.py

```diff
@@ -22,7 +22,7 @@
         "partner_identification",
         "partner_identification_gln",
         "base_edifact",
-        "edi_sale_order_import",
+        "edi_sale_oca",
```

Comment:
- actually here we just need to depend on `sale_order_import`, not `edi_sale_oca`
- in other words, `sale_order_import_edifact` just bring the capacity to import EDIFACT orders to `sale_order_import` wizard, and doesn't need the whole edi framework for that

Reply:
Yes, you're right, so we need to only depend on `sale_order_import`. Should I change the [`CONFIGURE.rst`](https://github.com/rmorant/edi/blob/80a630691dd63969c45474636c0056f195e63fc6/sale_order_import_edifact/readme/CONFIGURE.rst?plain=1#L2) too? And if changing, i should change only in v16 or v14 too?

Reply:
yes, please move remove that CONFIGURE.rst and move it to new module `edi_sale_edifact_oca`
only in v16 thanks

Reply:
Thank Nils, I updated